ci-operator/step-registry/telcov10n/sno-spoke/clean-up-hub-services/telcov10n-sno-spoke-clean-up-hub-services-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: telcov10n-sno-spoke-clean-up-hub-services
+  steps:
+    - ref: telcov10n-sno-spoke-cluster-clean-up
+    - ref: telcov10n-sno-spoke-gitops-clean-up
+    - ref: telcov10n-sno-spoke-site-config-clean-up
+    - ref: telcov10n-sno-spoke-gitea-clean-up
+  documentation: |-
+    These chainned steps allow to clean up the k8s services in the associated OCP Hub cluster
```

Comment:
Minor: "These chained..."

Question: Does it clean the hub or the spoke? Based on the step names, it appears to clean the spoke cluster (referring to the continuation of the documentation).

Reply:
The comment was intended to reflect that what these steps clean up are the objects that are created in the Hub cluster during the Spoke cluster deployment, such as the Gitea server that is created as a service in the Hub cluster and then used by the GitOps operator to read the Siteconfig and generate the objects that the ACM will process to perform the Spoke installation.

What was agreed was that, every time a new Spoke is to be deployed, as many artifacts as possible should be redeployed in order to be able to install it without reaching the point of having to redeploy the Hub and the operators associated with ZTP such as ACM and Gitops.

Reply:
Thanks for the clarification!